pandas/tests/io/formats/test_style.py

```diff
@@ -1215,11 +1215,7 @@ def test_highlight_max(self):
 
     def test_export(self):
         f = lambda x: "color: red" if x > 0 else "color: blue"
-        g = (
-            lambda x, y, z: "color: {z}".format(z=z)
-            if x > 0
-            else "color: {z}".format(z=z)
-        )
+        g = lambda x, y, z: f"color: {z}" if x > 0 else f"color: {z}"
```

Comment:
not related to the issue, maybe `y` can be removed here?

Reply:
Good idea! I'll look into taking care of it